modules/logging-upgrading-clo.adoc

```diff
@@ -6,28 +6,100 @@
 [id="logging-upgrading-clo_{context}"]
 = Updating the {clo}
 
-To update the {clo} to a new major release version, you must modify the update channel for the Operator subscription.
+The {clo} does not provide an automated upgrade from Logging 5.x to Logging 6.x because of the different combinations in which Logging can be configured. You must install all the different operators for managing logging seperately. You can upgrade to Logging 6.x version from both Logging 5.9 and Logging 5.8 versions.
+
+You can update {clo} by either changing the subscription channel in the {product-title} web console, or by uninstalling it. The following procedure demonstrates updating {clo} by changing the subscription channel in the {product-title} web console.
+
+[IMPORTANT]
+====
+When you migrate, all the logs that have not been compressed will be reprocessed by Vector. The reprocessing might lead to the following issues:
+
+* Duplicated logs during migration.
+* Too many requests in the Log storage receiving the logs, or requests reaching the rate limit.
+* Impact on the disk and performance because of reading and processing of all old logs in the collector.
+* Impact on the Kube API.
+* A peak in memory and cpu use by Vector until all the old logs are processed. The logs can be several GB per node. 
+====
 
 .Prerequisites
 
-* You have installed the {clo}.
+* You have updated the log collection and forwarding configurations to the link:https://github.com/openshift/cluster-logging-operator/blob/master/docs/reference/operator/api_observability_v1.adoc[`observability.openshift.io`] API.
 * You have administrator permissions.
-* You have access to the {ocp-product-title} web console and are viewing the *Administrator* perspective.
+* You have access to the {product-title} web console and are viewing the *Administrator* perspective.
 
 .Procedure
 
-. Navigate to *Operators* -> *Installed Operators*.
+. Create service account.
++
+If your previous forwarder is deployed in the namespace `openshift-logging` and named `instance`, then you have likely been using the service account `logcollector` created by earlier versions of the operator. In such a case you do not need to create a service account. However you must still assign RBAC permissions to the old service account as shown in the next step.
```

Comment:
That is wrong.
If your previous forwarder is deployed in the namespace `openshift-logging` and named `instance`, the service account `logcollector`  is created by earlier versions of the operator.  It is removed when you delete clusterlogging. You have to create the serviceaccount logcollector manualy in 6.x.

The other serviceaccount will be kept and can be used in 6.x directly.

Reply:
@anpingli thanks, does this mean that if users used a service account that they created manually in Logging 5.x, they don't need to create a service account and can instead just add RBAC permissions to it?

Reply:
Updated the text as follows:

If your previous log forwarder is deployed in the namespace `openshift-logging` and named `instance`, the earlier versions of the operator created a `logcollector` service account. This service account gets removed when you delete clusterlogging. Therefore, you need to create a new service account. Any other service account will be preserved and can be used in Logging 6.x. However, you must add Role-Based Access Control (RBAC) permissions to the service account.

WDYT?